geoips/create_plugin_registries.py

```diff
@@ -848,9 +849,30 @@ def main():
     args: list
         List of strings representing the arguments provided via command line.
     """
-    save_type = "json"
-    if len(sys.argv) > 1 and sys.argv[1].lower() == "yaml":
-        save_type = "yaml"
+    description = (
+        "Creates Plugin Registries for all installed GeoIPS packages. "
+        "The registries will be written to the root directory of each installed "
+        "package. The registries will be named either 'registered_plugins.json' "
+        "or 'registered_plugins.yaml' depending on which format is chosen. "
+        "For additional information on GeoIPS plugin registries please refer to "
+        "the GeoIPS documentation."
+    )
+
+    argparser = ArgumentParser(
+        prog="create_plugin_registries",
+        description=description,
+    )
+    argparser.add_argument(
+        "-s",
+        "--save_type",
+        type=str.lower,
+        default="json",
+        choices=["json", "yaml"],
+        help="Format to write registries to. This will also be the file extension.",
+    )
```

Comment:
There is an incoming change that allows passing a package name into create_plugin_registries - we will update this to allow a "-p" option when 1.12.1 is released.

Reply:
Sounds good. I think we can handle this when 1.12.1 comes through. I'm going to resolve and merge.